Remotion/Validation/UnitTests/RoleCollectors/AddingObjectValidationRuleCollector1Test.cs

```diff
@@ -46,54 +50,159 @@ public void SetUp ()
       _stubObjectValidator1 = new StubObjectValidator();
       _stubObjectValidator2 = new Mock<IObjectValidator>();
       _stubObjectValidator3 = new FakeCustomerValidator();
+      _validatedType = TypeAdapter.Create(typeof(Customer));
     }
 
     [Test]
-    [Ignore("TODO RM-5906")]
     public void Initialization_CollectorTypeDoesNotImplementIValidationRuleCollector_ThrowsArgumentException ()
     {
+      Assert.That(
+          () => new AddingObjectValidationRuleCollector<Customer>(typeof(Customer)),
+          Throws.ArgumentException
+              .With.ArgumentExceptionMessageEqualTo(
+                  "Parameter 'collectorType' is a 'Remotion.Validation.UnitTests.TestDomain.Customer', "
```

Comment:
```suggestion
                  "Parameter 'collectorType' is a 'Remotion.Validation.UnitTests.TestDomain.Customer' "
```

Reply:
I don't think that I should make this change. Then I would have to change it in `ArgumentUtility.cs` and also adapt all usages.